DevEdu.API/DevEdu.DAL/Models/CourseDto.cs

```diff
@@ -6,7 +6,7 @@ public class CourseDto : BaseDto
     {
         public string Name { get; set; }
         public string Description { get; set; }
-        public List<CourseTopicDto> CourseTopics { get; set; }
+        public List<TopicDto> Topics { get; set; }
```

Comment:
CourseTopicDto is not a simple relational entity, it has Position inside
so you can either use CourseTopicDto here or implement ordering inside the stored procedure
but anyway, Position of each topic must be returned from controller somehow =)

Reply:
Add to TopicDto